frontend/src/assets/lang/app/nl.json

```diff
@@ -226,7 +237,23 @@
     },
     "confirmations": {
         "cloneCourse": "Ben je zeker dat je dit vak wil klonen? Dit zal hetzelfde vak aanmaken voor het volgende academiejaar.",
-        "leaveCourse": "Ben je zeker dat je dit vak wil verlaten? Je zal geen toegang meer hebben tot dit vak."
+        "leaveCourse": "Ben je zeker dat je dit vak wil verlaten? Je zal geen toegang meer hebben tot dit vak.",
+        "shareCourse": "Door het creëren van een invitatielink staat u studenten in bezit van deze link toe zich in te schrijven voor dit vak. Gelieve de gegenereerde link te kopiëren, pas wanneer u op \"Creëer invitatielink\" klikt zal deze link geldig worden."
+    },
+    "protectedCourses": {
+        "screen1": {
+            "title": "Bemachtigen link",
+            "content": "Professoren kunnen kiezen om hun vakken niet publiek te maken. Vraag de prof om een invitatielink te delen om te kunnen toetreden tot het vak."
+        },
+        "screen2": {
+            "title": "Vak zoeken",
+            "content": "Gebruik de link om het vak te zoeken. Als je het vak niet kan vinden, kan je de prof vragen om een nieuwe link te delen."
+
+        },
+        "screen3": {
+            "title": "Inschrijven",
+            "content": "Schrijf je in voor het vak. Je kan nu alle lopende projecten raadplegen, deadlines, ..."

```

Comment:
Weird to write comments about Dutch text, but maybe it's better to put "raadplegen" after the non-exhaustive listing. So that would be "projecten, deadlines, ... raadplegen". Because now it seems "raadplegen" is only talking about the projects, but not the deadlines and all the rest.